mlir-sycl/test/Conversion/SYCLToSPIRV/grid-ops.mlir

```diff
@@ -13,17 +13,17 @@ module attributes {gpu.container_module} {
 
```

Comment:
This test was passing because `CHECK-NEXT` tags were missing `:`.

Reply:
Wow we all miss that mistake :-(